environment_dev.yaml

```diff
@@ -16,6 +16,7 @@ dependencies:
 - networkx>=1.10
 - pyomo>=5.7.0,<6.6.2
 - cartopy>=0.16
+- geopandas>=0.9
```

Comment:
Should `shapely` be added here for the development environment, since it is used in unit tests?

Reply:
already included in geopandas dependencies